client/src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import '../../../node_modules/bulma/css/bulma.css';
+import moment from 'moment';
+
+import styled from 'styled-components';
+import './Calendar.css'
+
+import { Week } from '../';
+
+// start with sunday start. focus on monday start after
+// something with the fact that there are two divs of columns
+const CalContainer = styled.div`
+  height: 45vh;
+  overflow: scroll;
+`;
+
+const DateBox = styled.div`
+  background-color: #2E2F2F;
+  margin: 2px;
+  padding: 5px;
+  color: #CDDDDD;
+
+  &:hover {
+    background-color: ${props => props.hasModal && '#4A4C4C'};
+  }
+`;
+
+const FirstBoxKpi = styled.div`
+  background-color: #2E2F2F;
+  margin: 2px;
+  margin-left: 10px;
+  color: #CDDDDD;
+`;
+
+const H3 = styled.div`
+  text-align: center;
+`;
+  
+function Calendar({ user, runData }) {  
+
+
+  const weekDayNums = Array.of(0, 1, 2, 3, 4, 5, 6);  
+
+  return (
+    <div>
+      <div className="columns is-mobile calendar">
+        {
+          weekDayNums.map(el => {
```

Comment:
If you don't have additional logic, you can save some boilerplate by doing and automatic return:

```js
weekDayNums.map(el => (
  <DateBox className="column" key={el}>
    <H3>{`${moment.weekdaysShort(el)}`}</H3>
  </DateBox>
))
```